MemoizR/EagerRelativeSignal.cs

```diff
@@ -32,7 +32,7 @@ public async Task Set(Func<T?, T> fn)
         }
     }
 
-    public async Task<T?> Get()
+    public async ValueTask<T?> Get()
     {
```

Comment:
I do not want to expose ValueTask here as it is expected that Get() will be evaluated multiple times.

Reply:
> Valid consumption patterns for ValueTasks
> Awaiting a ValueTask / ValueTask<TResult> multiple times. The underlying object may have been recycled already and be in use by another operation. In contrast, a Task / Task<TResult> will never transition from a complete to incomplete state, so you can await it as many times as you need to, and will always get the same answer every time.
https://devblogs.microsoft.com/dotnet/understanding-the-whys-whats-and-whens-of-valuetask/

Reply:
Fair - One might pass down a Signal down the component tree in a UI Framework.